js/global_variables/cell_exp_array.js

```diff
@@ -16,28 +16,22 @@ export const update_cell_exp_array = async (cats, genes, base_url, inst_gene, ve
     let cell_exp = exp_table.getChild(inst_gene).toArray()
 
     const new_exp_array = new Array(cats.cell_names_array.length).fill(0)
-
+    
     cell_names.forEach((name, i) => {
-        name = String(name)
-        //console.log(name, typeof name)
-
-        if (cats.cell_name_to_index_map.has(name)) {
-            const index = cats.cell_name_to_index_map.get(name);
+        if (name in cats.nameMapping_inv) {
             const exp_value = Number(cell_exp[i]);
-
             const max_exp = Number(genes.meta_gene[inst_gene].max);
 
             // Apply logarithmic transformation
             const log_exp_value = Math.log1p(exp_value); // log1p(x) = log(1 + x)
 
             // Scale to 0-255 range using the log of the max expression value
             const log_max_exp = Math.log1p(max_exp); // log1p(x) = log(1 + x)
-            new_exp_array[index] = (log_exp_value / log_max_exp) * 255;
+            new_exp_array[name] = (log_exp_value / log_max_exp) * 255;
         } else {
-            console.log('Cell name not found in cell_name_to_index_map');
+            console.log('Cell name not found in cats.nameMapping_inv');
```

Comment:
lets remove this console log since it slows down the browser if there is an error.